### PR TITLE
Revise porphyrin metabolism disease hierarchy

### PR DESCRIPTION
**Pull request by twhetzel**
closes https://github.com/monarch-initiative/mondo/issues/7381

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -585022,6 +585026,21 @@ is_a: MONDO:0004981 {source="PMID:38033089"} ! atrial fibrillation
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7886" xsd:anyURI
 
+[Term]
+id: MONDO:1030015
+name: acquired porphyria
+def: "A porphyria disorder that is acquired during the lifetime of the individual." [MONDO:patterns/acquired]
+xref: GARD:10353 {source="MONDO:GARD"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:585033]: This x-ref cannot be from MONDO:GARD. MONDO:GARD means that GARD told us that it is the correct x-ref. Here, you said it was, not GARD. You should add a comment and CC GARD (ie Eric Sid) and let him know about this x-ref.
 
That being said, this x-ref refers to porphyria and not acquired porphyria. It should not be on this term. Was there a reason to add it here?

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -585022,6 +585026,21 @@ is_a: MONDO:0004981 {source="PMID:38033089"} ! atrial fibrillation
...
+xref: ICD9:277.1 {source="MONDO:equivalentTo"}
+xref: MESH:D011164 {source="MONDO:relatedTo"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:585035]: This x-ref refers to porphyria. If you want to add it here, the correct source would be MONDO:mondoIsNarrowerThanSource (and not MONDO:relatedTo)

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -585022,6 +585026,21 @@ is_a: MONDO:0004981 {source="PMID:38033089"} ! atrial fibrillation
...
+xref: NCIT:C97096 {source="MONDO:relatedTo"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:585036]: This x-ref refers to porphyria. If you want to add it here, the correct source would be MONDO:mondoIsNarrowerThanSource (and not MONDO:relatedTo)

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -585022,6 +585026,21 @@ is_a: MONDO:0004981 {source="PMID:38033089"} ! atrial fibrillation
...
+xref: SCTID:154768004 {source="MONDO:equivalentTo"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:585037]: I cannot find this x-ref in snomed. Is it the correct one? 
Also note that there are 2 SCTID which are MONDO:equivalentTo

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -585022,6 +585026,21 @@ is_a: MONDO:0004981 {source="PMID:38033089"} ! atrial fibrillation
...
+xref: SCTID:86292002 {source="MONDO:equivalentTo"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:585038]: I cannot find this x-ref in snomed. Is it the correct one?
Also note that there are 2 SCTID which are MONDO:equivalentTo

File: src/ontology/mondo-edit.obo [hunk 6]
```diff
@@ -585022,6 +585026,21 @@ is_a: MONDO:0004981 {source="PMID:38033089"} ! atrial fibrillation
...
+is_a: MONDO:0037939 ! porphyria
+intersection_of: MONDO:0037939 ! porphyria
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:585040]: This SC annotation needs a source

File: src/ontology/mondo-edit.obo [hunk 7]
```diff
@@ -432559,9 +432560,9 @@ xref: ICD9:277.1 {source="DOID:13268"}
 xref: MedDRA:10036181 {source="Orphanet:738", source="Orphanet:738/e"}
 xref: MedDRA:10061356 {source="Orphanet:738", source="Orphanet:738/e"}
 xref: MEDGEN:698423 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-xref: MESH:D011164 {source="MONDO:equivalentTo", source="Orphanet:738", source="DOID:13268", source="Orphanet:738/e"}
+xref: MESH:D011164 {source="MONDO:relatedTo", source="Orphanet:738", source="DOID:13268", source="Orphanet:738/e"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:432563]: This x-ref refers to porphyria. If you want to add it here, the correct source would be MONDO:mondoIsNarrowerThanSource (and not MONDO:relatedTo).

File: src/ontology/mondo-edit.obo [hunk 8]
```diff
@@ -432559,9 +432560,9 @@ xref: ICD9:277.1 {source="DOID:13268"}
...
 xref: NANDO:2200610 {source="MONDO:NANDO", source="https://orcid.org/0000-0003-0011-764X", source="https://orcid.org/0000-0002-0170-9172"}
-xref: NCIT:C97096 {source="MONDO:equivalentTo", source="DOID:13268"}
+xref: NCIT:C97096 {source="MONDO:relatedTo", source="DOID:13268"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:432565]: This x-ref refers to porphyria. If you want to add it here, the correct source would be MONDO:mondoIsNarrowerThanSource (and not MONDO:relatedTo).

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@twhetzel please see comments in line. 
Also, based on our discussions, I expected to also see a term for the  “acquired” form of disorder of porphyrin metabolism. Is there a reason why you didn't add it? (there could be some reasons. If so, it would be good to document it here)

File: src/ontology/mondo-edit.obo (hunk 7 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:432563]: right, that's the one i meant to add. fixed

File: src/ontology/mondo-edit.obo (hunk 8 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:432565]: fixed

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:585035]: fixed

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:585036]: fixed

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:585033]: I was reviewing the xrefs for 'inherited porphyria' and seeing which ones were general for porpyria and could be added as MONDO:mondoIsNarrowerThanSource so this new term would have some connections to other vocabs.

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:585037]: https://termbrowser.nhs.uk/?perspective=full&conceptId1=154768004&edition=uk-edition&release=v20240731&server=https://termbrowser.nhs.uk/sct-browser-api/snomed&langRefset=999001261000000100,999000691000001104

Comment (twhetzel) [reply src/ontology/mondo-edit.obo:585037]: changed to MONDO:mondoIsNarrowerThanSource

File: src/ontology/mondo-edit.obo (hunk 5 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:585038]: https://termbrowser.nhs.uk/?perspective=full&conceptId1=86292002&edition=uk-edition&release=v20240731&server=https://termbrowser.nhs.uk/sct-browser-api/snomed&langRefset=999001261000000100,999000691000001104

and changed to MONDO:mondoIsNarrowerThanSource

File: src/ontology/mondo-edit.obo (hunk 6 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:585040]: added

File: src/ontology/mondo-edit.obo (hunk 5 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:585038]: this ID is inactive from what I can tell (see "details" in the link you sent).  I don't think we should use it as an x-ref.

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:585037]: this ID is inactive from what I can tell (see "details" in the link you sent).  I don't think we should use it as an x-ref.

File: src/ontology/mondo-edit.obo [hunk 9]
```diff
@@ -585022,6 +585026,21 @@ is_a: MONDO:0004981 {source="PMID:38033089"} ! atrial fibrillation
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7886" xsd:anyURI
 
+[Term]
+id: MONDO:1030015
+name: acquired porphyria
+def: "A porphyria disorder that is acquired during the lifetime of the individual." [MONDO:patterns/acquired]
+xref: GARD:10353 {source="MONDO:mondoIsNarrowerThanSource"}
+xref: ICD9:277.1 {source="MONDO:equivalentTo"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:585034]: Sorry I missed it previously. According to the file found [here](https://www.cms.gov/medicare/coding-billing/icd-10-codes/icd-9-cm-diagnosis-procedure-codes-abbreviated-and-full-code-titles#:~:text=Version%2032%20Full%20and%20Abbreviated%20Code%20Titles%20%C2%A0%E2%80%93%20Effective%20October%201%2C%202014%20(ZIP)), this ICD9 code is not specific to the acquired disease

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:585033]: I am ok with MONDO:mondoIsNarrowerThanSource

File: src/ontology/mondo-edit.obo (hunk 6 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:585040]: GARD:20353 is not an exact match, so we should not use it as a source for this SC. 
Is there a PMID where they mention that there are acquired forms of porphyria? Maybe a review?

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@twhetzel 
I have a few additional change request (and I just now realized that this PR might not have been ready for me yet - sorry! )

please note my previous question about a term for the “acquired” form of disorder of porphyrin metabolism.

File: src/ontology/mondo-edit.obo (hunk 9 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:585034]: I removed it from here and added it to 'porphyrin metabolism disease', which is appropriate based on the UMLS xref already on that term.

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:585037]: removed

File: src/ontology/mondo-edit.obo (hunk 5 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:585038]: removed

**Review by sabrinatoro — APPROVED** [review]
Looks good. Approving and merging.

**Comment by twhetzel** [comment]
@sabrinatoro I realized after we talked the 'acquired' form of a disease (MONDO:0008295 'sporadic porphyria cutanea tarda') I came across is actually a 'porphyria' and not a 'inborn disorder of porphyrin metabolism'.

**Pull request merged**